ModelReplacementAPI/Modules/AvatarUpdater.cs

```diff
@@ -14,48 +14,46 @@ public class AvatarUpdater
 
-        public Vector3 itemHolderPositionOffset { get; private set; } = Vector3.zero;
-        public Quaternion itemHolderRotationOffset { get; private set; } = Quaternion.identity;
-        public Transform itemHolder { get; private set; } = null;
+        public Vector3 ItemHolderPositionOffset { get; private set; } = Vector3.zero;
+        public Quaternion ItemHolderRotationOffset { get; private set; } = Quaternion.identity;
+        public Transform ItemHolder { get; private set; } = null;
 
         protected bool hasUpperChest = false;
-        protected Vector3 rootPositionOffset = new Vector3(0, 0, 0);
-        protected Vector3 rootScale = new Vector3(1, 1, 1);
+        protected Vector3 rootPositionOffset = Vector3.zero;
+        protected Vector3 rootScale = Vector3.one;
 
         public virtual void AssignModelReplacement(GameObject player, GameObject replacement)
         {
             PlayerControllerB controller = player.GetComponent<PlayerControllerB>();
-            if (controller)
-            {
-                playerModelRenderer = controller.thisPlayerModel;
-            }
-            else
-            {
-                playerModelRenderer = player.GetComponentInChildren<SkinnedMeshRenderer>();
-            }
+            playerModelRenderer = controller ? controller.thisPlayerModel : player.GetComponentInChildren<SkinnedMeshRenderer>();
 
             if (playerModelRenderer == null)
             {
-                ModelReplacementAPI.Instance.Logger.LogFatal("failed to start AvatarBodyUpdater");
+                ModelReplacementAPI.Instance.Logger.LogFatal("Failed to start AvatarBodyUpdater");
                 return;
             }
-            this.player = player;
 
+            this.player = player;
             replacementAnimator = replacement.GetComponentInChildren<Animator>();
             this.replacement = replacement;
 
-            OffsetBuilder ite = replacementAnimator.gameObject.GetComponent<OffsetBuilder>();
-            itemHolderPositionOffset = ite.itemPositonOffset;
-            itemHolderRotationOffset = ite.itemRotationOffset;
-            itemHolder = ite.itemHolder.transform;
-            rootPositionOffset = ite.rootPositionOffset;
-            rootScale = ite.rootScale;
+            OffsetBuilder offsetBuilder = replacementAnimator.gameObject.GetComponent<OffsetBuilder>();
+            ItemHolderPositionOffset = offsetBuilder.itemPositonOffset;
+            ItemHolderRotationOffset = offsetBuilder.itemRotationOffset;
+            ItemHolder = offsetBuilder.itemHolder.transform;
+            rootPositionOffset = offsetBuilder.rootPositionOffset;
+            rootScale = offsetBuilder.rootScale;
+
             Vector3 baseScale = replacement.transform.localScale;
-            replacement.transform.localScale = new Vector3(1, 0, 0) * baseScale.x * rootScale.x + new Vector3(0, 1, 0) * baseScale.y * rootScale.y + new Vector3(0, 0, 1) * baseScale.z * rootScale.z;
```

Comment:
There's actually a much better way of doing this. `Vector3.Scale`, I will include that as a component of this refactor when I pull it.